frontend/src/v5/ui/routes/dashboard/projects/federations/createFederationForm/createFederationForm.component.tsx

```diff
@@ -99,7 +95,7 @@ export const CreateFederationForm = ({ open, onClickClose }: ICreateFederation):
 		<FormModal
```

Comment:
Not specific to this PR, but if I try to create a federation and the name already exists, I get an error modal instead of being brought back to the modalPhase `settings` and receiving the error `This name already exists` right below the input textfield. Also, the name is not trimmed, so I can insert a name made up of blank spaces only, which I am not sure should be allowed